app.js

```diff
@@ -65,6 +66,99 @@ app.get('/logout', (req, res) => {
   res.redirect('/');
 });
 
+// renders profile page with customer info and pets
+app.get("/customerProfile", function (req, res){
+  // get customer
+  let custQuery = "SELECT * FROM customers WHERE customer_id = ?";
+  let custParams = [req.customerId];
+  let customer = pool.query(custQuery, custParams, function(err, rows, fields){
+    if(err) throw err;
+    console.log(err);
+  });
+  
+  // get pets
+  let petQuery = "SELECT * FROM pets WHERE customer_id = ?";
+  let pets = pool.query(petQuery, custParams, function(err, rows, fields){
+    if(err) throw err;
+    console.log(err);
+  });
+  
+  req.render("customerProfile", {"customer": customer, "pets": pets});
+});// customerProfile
+
+// updates and returns customer (no pet info)
+app.put("api/updateProfile", function (req, res){
+  
+  let custQuery = "UPDATE customers WHERE customer_id = SET first_name = ?, last_name = ?, email = ?, phone = ?, address_1 = ?, address_2 = ?, city = ?, sate = ?, zip = ?";
+  let custParam = [req.customerId, req.query.fn, req.query.ln, req.query.email, req.query.phone, req.query.addr1, req.query.addr2, req.query.city, req.query.state, req.query.zip];
+  
+  // returns updated customer
+  pool.query(custQuery, custParam, function (err, rows, fields){
+    if(err) throw err;
+    console.log(rows);
+    res.send(rows);
+  });  
+  
+});// api/updateProfile
+
+// renders service page with pets and services
+app.get("/scheduleService", function(req, res){
+  // get pets
+  let petQuery = "SELECT * FROM pets WHERE customer_id = ?";
+  let petParams = [req.customerId];
+  let pets = pool.query(petQuery, petParams, function(err, rows, fields){
+    if(err) throw(err);
+    console.log(rows);
+  });
+  
+  // get services
+  let serviceQuery = "SELECT * FROM services";
+  let services = pool.query(serviceQuery, function(err, rows, fields){
```

Comment:
I don't think these are going to work since `pool.query` is asynchronous right? So we won't get a return values from these queries right away.